publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/helper/RabbitMqProperties.java

```diff
@@ -440,4 +467,24 @@ private Channel giveMeRandomChannel() {
         return rabbitChannels.get(random.nextInt(rabbitChannels.size()));
     }
 
+    /**
+     * This method is used to get routing key type based on the eventType from the configuration file
+     * 
+     * @param eventType
+     *            Eiffel eventType
+     * @return type based on eventType if provided in the configuration file else null
+     */
+    public String getTypeRoutingKeyFromConfiguration(String eventType) {
+        if (types != null) {
+            String key = eventType + DOT + TYPE;
+            try {
+                if (!types.getString(key).isEmpty()) {
```

Comment:
Why to perform the same code twice? I'd store result of types.getString(key) into a variable and reuse it on the next line.

Reply:
Added